packages/api/src/routes/v3/report.ts

```diff
@@ -7,6 +7,7 @@ interface PostReportBody {
   readonly url: string;
   readonly userAgent?: string;
   readonly version: string;
+  readonly explanation?: string;
```

Comment:
Please, order the properties alphabetically.

Reply:
Done!